test/regress/regress0/error.cvc

```diff
@@ -1,8 +1,7 @@
+% ERROR-SCRUBBER: sed -e '/^[[:space:]]*$/d'
```

Comment:
Wouldn't it be easier to just remove empty lines in general? Also, you mention that test cases were failing due to line ending differences but I can't seem to find changes addressing that. What am I missing? :)